packages/appstore/src/components/modals/account-type-modal/account-type-modal.tsx

```diff
@@ -54,25 +44,35 @@ const ModalContent = ({
     selectAccountTypeCard,
     is_financial_available,
     is_synthetic_available,
+    is_swapfree_available,
 }: TModalContent) => {
     return (
         <div className='account-type-card__wrapper'>
             {is_synthetic_available && (
                 <AccountCard
                     account_type_card={account_type_card}
-                    selectAccountTypeCard={() => selectAccountTypeCard(`${derived_account.title_and_type}`)}
-                    description={derived_account.description}
-                    title_and_type={derived_account.title_and_type}
-                    icon={derived_account.icon}
+                    selectAccountTypeCard={() => selectAccountTypeCard(`${getDerivedAccount().title_and_type}`)}
```

Comment:
You can create an array with the values ['synthetic', 'financial', 'swapfree'].map(acc_type) and use acc_type to get rid of the repetetive code.

Reply:
we need to still check whether the account is available or not?
is_swapfree_available etc.